apps/src/templates/SchoolAutocompleteDropdown.jsx

```diff
@@ -42,7 +43,10 @@ export default class SchoolAutocompleteDropdown extends Component {
    *   returns results or a request error occurs.
    */
   debouncedSearch = _.debounce((q, callback) => {
-    const searchUrl = `/dashboardapi/v1/schoolsearch/${encodeURIComponent(q)}/40`;
+    const searchUrl = `/dashboardapi/v1/schoolsearch/${encodeURIComponent(q)}/40` +
+      (experiments.isEnabled(experiments.SCHOOL_AUTOCOMPLETE_DROPDOWN_NEW_SEARCH) ?
+       '/true' : '');
```

Comment:
API design is bit strange - I'd expect `/enableThisThing` instead of `/true`, but for purposes of this PR, it seems like you've added what you need.

Reply:
I believe that the string can have any value. I agree that the effect would be clearer if it were something like "useNewSearch" rather than "true".